### PR TITLE
feat: media details

### PR DESCRIPTION
**Pull request by seferturan**
## 🎶 Notes 🎶

- I'm not the biggest fan of using links for things that are not a link. What do you think about using a button instead @vladjerca? See below for an example.
- This is the first version; will need follow-ups to display shows specific data & director/writer data.

## 👀 Example 👀
Screenshot 2024-12-22 at 20 54 45

### CODE REVIEW

File: projects/client/src/lib/utils/formatting/date/toHumanDay.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import type { AvailableLocale } from '$lib/features/i18n/index.ts';
+import { LOCALE_MAP } from '$lib/utils/formatting/date/LOCALE_MAP.ts';
+import { format } from 'date-fns/format';
+
+export function toHumanDay(
+  date: Date,
```
Comment (vladjerca) [inline projects/client/src/lib/utils/formatting/date/toHumanDay.ts:6]: Tests for this

File: projects/client/src/lib/sections/summary/utils/constructMediaDetails.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const genres = media.genres.map(i18n.genre);
+  const studioNames = studios.map((studio) => studio.name);
+
+  const languages = media.languages?.map((language) =>
+    toLanguageName(language, languageTag())
+  );
+
+  return [
+    {
+      title: 'Premiered',
```
Comment (vladjerca) [inline projects/client/src/lib/sections/summary/utils/constructMediaDetails.ts]: Why do we hardcode these titles in English?

File: projects/client/src/lib/utils/formatting/intl/_internal/toDisplayName.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+type ToDisplayNameProps = {
+  value: string;
+  languageTag: AvailableLanguage;
+  type: DisplayNamesType;
+};
+
+export function toDisplayName({
+  value,
+  languageTag,
```
Comment (vladjerca) [inline projects/client/src/lib/utils/formatting/intl/_internal/toDisplayName.ts:13]: Tests for this

File: projects/client/src/lib/utils/formatting/intl/toCountryName.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import type { AvailableLanguage } from '$lib/features/i18n/index.ts';
+import { toDisplayName } from '$lib/utils/formatting/intl/_internal/toDisplayName.ts';
+
+export function toCountryName(value: string, languageTag: AvailableLanguage) {
```
Comment (vladjerca) [inline projects/client/src/lib/utils/formatting/intl/toCountryName.ts]: Tests for this

File: projects/client/src/lib/utils/formatting/intl/toLanguageName.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import type { AvailableLanguage } from '$lib/features/i18n/index.ts';
+import { toDisplayName } from '$lib/utils/formatting/intl/_internal/toDisplayName.ts';
+
+export function toLanguageName(value: string, languageTag: AvailableLanguage) {
```
Comment (vladjerca) [inline projects/client/src/lib/utils/formatting/intl/toLanguageName.ts]: Tests for this

File: projects/client/src/lib/sections/summary/utils/constructMediaDetails.ts (hunk 2 above)
Comment (seferturan) [reply projects/client/src/lib/sections/summary/utils/constructMediaDetails.ts]: Because I forgot 😅

**Pull request merged**